packages/core/package.json

```diff
@@ -32,6 +32,7 @@
     "homepage": "https://github.com/binary-com/deriv-app",
     "devDependencies": {
         "@babel/eslint-parser": "^7.17.0",
+        "@babel/plugin-proposal-private-property-in-object": "^7.12.1",
```

Comment:
can we use `@babel/plugin-transform-private-property-in-object` instead?
![image](https://github.com/binary-com/deriv-app/assets/100833613/60ef6ce9-f6ab-4a7a-aede-c50b0eba37d8)

Reply:
@shayan-deriv thanks for mentioning this but if I want to replace this I have to change 13 files and it's out of the scope of this card. will do that in a separate PR 🙏